cypress/integration/projectSettings/project_settings.ts

```diff
@@ -2,56 +2,54 @@ import {
   getAccessRoute,
   getGeneralRoute,
   getGithubCommitQueueRoute,
+  getVirtualWorkstationRoute,
   project,
   projectUseRepoEnabled,
   repo,
   saveButtonEnabled,
 } from "./constants";
 import { clickSave } from "../../utils";
 
-describe("Access page", { testIsolation: false }, () => {
-  const destination = getAccessRoute(projectUseRepoEnabled);
-  before(() => {
-    cy.visit(destination);
-  });
-
-  it("Save button should be disabled on initial load", () => {
+describe("Access page", () => {
+  const origin = getAccessRoute(projectUseRepoEnabled);
+  beforeEach(() => {
+    cy.visit(origin);
     saveButtonEnabled(false);
-  });
-
-  it("Shows a 'Default to Repo on Page' button on page", () => {
-    cy.dataCy("default-to-repo-button").should("exist").should("be.enabled");
+    cy.dataCy("default-to-repo-button")
+      .should("be.visible")
+      .should("be.enabled");
   });
 
   it("Changing settings and clicking the save button produces a success toast and the changes are persisted", () => {
-    cy.getInputByLabel("Unrestricted").parent().click();
+    // Wait for internal access buttons to load
+    cy.get("[aria-label='Internal Access']")
+      .find("label")
+      .as("internalAccessButtons")
+      .should("have.length", 3);
+    cy.contains("label", "Unrestricted").click();
     cy.getInputByLabel("Unrestricted").should(
       "have.attr",
       "aria-checked",
       "true"
     );
-
+    // Input and save username
     cy.contains("Add Username").click();
-    cy.get("[aria-label='Username'")
-      .should("have.length", 1)
-      .first()
-      .type("admin");
-    cy.get("[aria-label='Username']")
-      .should("have.value", "admin")
-      .should("exist");
+    cy.get("[aria-label='Username'").as("usernameInput").type("admin");
```

Comment:
We have a method for this called `cy.getInputByLabel` https://github.com/evergreen-ci/spruce/blob/c80310a696663d5d1cc1ec45506f26b83849153a/cypress/support/e2e.ts#L79